src/unicode.js

```diff
@@ -120,22 +231,114 @@ const halfChars = [
   'í',
 ];
 
+const aboveChars = [
+  'ੇ',
+  'ੈ',
+  'ੋ',
+  'ੌ',
+];
+
+/**
+ * Convert Gurmukhi Unicode to ascii for webakhar
+ *
+ * @since 1.0.0
+ * @param {string} unicode text script to be converted
+ * @returns {string} Returns ascii text
+ * @example
+ *
+ * unicode('ਆਇ ਮਿਲੁ ਗੁਰਸਿਖ ਆਇ ਮਿਲੁ ਤੂ ਮੇਰੇ ਗੁਰੂ ਕੇ ਪਿਆਰੇ ॥');
+ * // => 'Awie imlu gurisK Awie imlu qU myry gurU ky ipAwry ]'
+ */
+
+function ascii(text = '') {
+  const convertedText = [];
+
+  const chars = text.split('');
+
+  for (let j = 0; j < chars.length; j += 1) {
+    const currentChar = chars[j];
+    const nextChar = chars[j + 1];
+    const nextNextChar = chars[j + 2];
+
+    if (currentChar === 'ਿ') {
+      const lastElement = convertedText.pop();
+      convertedText.push('i');
+      convertedText.push(lastElement);
+    } else if (currentChar === '੍') {
+      if (nextNextChar === 'ਿ') {
+        const lastElement = convertedText.pop();
+        convertedText.push('i');
+        convertedText.push(lastElement);
+        j += 1;
+      }
+      convertedText.push(reverseMapping[currentChar + nextChar]);
+      if (nextNextChar === 'ੁ') {
+        convertedText.push('ü');
+        j += 1;
+      } else if (nextNextChar === 'ੂ') {
+        convertedText.push('¨');
+        j += 1;
+      }
+      j += 1;
+    } else if (currentChar === 'ੑ') {
+      convertedText.push(reverseMapping[currentChar] || currentChar);
+      if (nextChar === 'ੁ') {
+        convertedText.push('ü');
+        j += 1;
+      } else if (nextChar === 'ੂ') {
+        convertedText.push('¨');
+        j += 1;
+      }
+    } else if (currentChar === 'ਨ' && nextChar === 'ੂ' && nextNextChar === 'ੰ') {
+      convertedText.push('ƒ');
+      j += 2;
+    } else if (currentChar === 'ੋ' && nextChar === 'ੁ') {
+      convertedText.push(reverseMapping[currentChar + nextChar]);
+      j += 1;
+    } else if (currentChar === 'ੱ' && aboveChars.includes(nextNextChar)) {
+      convertedText.push('¤');
+    } else if (currentChar === 'ਾ' && nextChar === 'ਂ') {
+      convertedText.push('W');
+      j += 1;
+    } else if (currentChar === 'ਆ' && nextChar === 'ਂ') {
+      convertedText.push('AW');
+      j += 1;
+    } else if (
+      (currentChar === 'ਂ' && nextChar === 'ੀ')
+      || (currentChar === 'ੀ' && nextChar === '')
+    ) {
+      convertedText.push('ˆØI');
+      j += 1;
+    } else {
+      convertedText.push(reverseMapping[currentChar] || currentChar);
+    }
+  }
+
+  return convertedText.join('');
+}
+
 /**
  * Convert Gurmukhi script to Unicode
  *
  * @since 1.0.0
- * @param {string} text Gurbani Akhar script to be converted
+ * @param {string} text Gurbani Akhar or unicode script to be converted
+ * @param {boolean} do a unicode to ascii (true) or not (false)
```

Comment:
can you add an example here and to the `README.md` as well.

Reply:
example is below your comment

Reply:
An example where reverse is passed as true?

line 249 has it but without the second parmeter. 
```
unicode('ਆਇ ਮਿਲੁ ਗੁਰਸਿਖ ਆਇ ਮਿਲੁ ਤੂ ਮੇਰੇ ਗੁਰੂ ਕੇ ਪਿਆਰੇ ॥');
 * // => 'Awie imlu gurisK Awie imlu qU myry gurU ky ipAwry ]'
```
can we add it to README?

Reply:
done